src/openrct2/rct1/Tables.cpp

```diff
@@ -696,8 +696,12 @@ namespace RCT1
         return map[vehicleSubEntry];
     }
 
-    std::string_view GetRideTypeObject(RideType rideType)
+    std::string_view GetRideTypeObject(RideType rideType, bool isLL)
     {
+        if (rideType == RideType::InvertedRollerCoaster && !isLL) {
+            return "rct1.ride.inverted_trains";
```

Comment:
Ditto, why can't we use `RCT1::GetRideTypeObject`?

Reply:
I don’t understand. This **is** `RCT1::GetRideTypeObject`.

Reply:
Oh, my bad!